packages/shared-components/src/template/templates/navdesign/wizardHeader/form.ejs

```diff
@@ -1,53 +1,53 @@
+<div ref="{{ctx.wizardKey}}-header">
+    <button class="stepper-toggle navds-button navds-button--secondary navds-button--small" ref="{{ctx.wizardKey}}-stepper-open">
+        <svg width="1em" height="1em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" role="img">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="m6.586 12 7.707-7.707 1.414 1.414L9.414 12l6.293 6.293-1.414 1.414L6.586 12Z" fill="currentColor" />
+        </svg>
+        Steg {{ctx.currentPage + 1}} av {{ctx.panels.length + 1}}
+    </button>
+    <div class="stepper-backdrop" ref="{{ctx.wizardKey}}-stepper-backdrop"></div>
+    <nav aria-label="Søknadssteg" id="{{ ctx.wizardKey }}-header" class="stegindikator stepper" ref="{{ctx.wizardKey}}-stepper">
+        <div class="stepper-container">
+            <button class="stepper-close navds-button navds-button--tertiary navds-button--medium" ref="{{ctx.wizardKey}}-stepper-close">
+                <span class="navds-button__icon">
+                    <svg width="1em" height="1em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" role="img" aria-hidden="true">
+                        <title>{{ctx.t("Lukk")}}</title>
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M21 4.385 13.385 12 21 19.615 19.615 21 12 13.385 4.385 21 3 19.615 10.615 12 3 4.385 4.385 3 12 10.615 19.615 3 21 4.385Z" fill="currentColor"></path>
+                    </svg>
+                </span>
+                {{ctx.t("Lukk")}}
+            </button>
+            <ol class="navds-stepper">
+                {% ctx.panels.forEach(function(panel, index) { %}
+                <li class="navds-stepper__item">
+                    <span class="navds-stepper__line navds-stepper__line--1"></span>
+                    <a
+                        href="/fyllut/{{ctx.form.path}}/{{panel.key}}"
+                        class="navds-stepper__step {{ctx.currentPage > index ? 'navds-stepper__step--behind navds-stepper__step--completed' : ctx.currentPage === index ? 'navds-stepper__step--active' : ''}}"
+                        ref="{{ctx.wizardKey}}-link"
+                        {% if (ctx.currentPage === index) { %}
+                        aria-current="step"
+                        {% } %}
+                    >
+                        <span class="navds-stepper__circle navds-label" aria-hidden="true">{{ctx.t(index+1)}}</span>
+                        <span class="navds-stepper__content navds-label">{{ctx.t(ctx.panels[index].title, { _userInput: true })}}</span>
+                    </a>
+                    <span class="navds-stepper__line navds-stepper__line--2"></span>
+                </li>
+                {% }) %}
+                <li class="navds-stepper__item">
+                    <span class="navds-stepper__line navds-stepper__line--1"></span>
+                    <a
+                            href="/fyllut/{{ctx.form.path}}/oppsummering"
```

Comment:
fungerer dette i byggeren?

Reply:
Det fungerer, fordi det er onClick-eventer som "overstyrer" navigeringen. Bytter likevel til href="#" for å tydeliggjøre at URL-en ikke har noe å si i praksis, selv om jeg mener det hadde vært bedre å synliggjøre for sluttbruker hvor URL-en leder.